packages/engine/src/avatar/systems/AvatarAnimationSystem.ts

```diff
@@ -195,9 +191,13 @@ const execute = () => {
 
   const avatarAnimationQueryArr = avatarAnimationQuery()
   const avatarAnimationEntities: Entity[] = []
-  for (let i = 0; i < avatarAnimationQueryArr.length; i++) {
-    if (filterPriorityEntities(avatarAnimationQueryArr[i])) avatarAnimationEntities.push(avatarAnimationQueryArr[i])
-  }
+
+  avatarAnimationQueryArr.forEach((_entity) => {
```

Comment:
`for (let i; ...)` is far faster than .forEach,  we need to keep the original syntax here